jetty-hazelcast/src/main/java/org/eclipse/jetty/hazelcast/session/HazelcastSessionDataStoreFactory.java

```diff
@@ -171,6 +173,7 @@ public SessionDataStore getSessionDataStore(SessionHandler handler)
                             LOG.warn("Hazelcast xml config is missing org.eclipse.jetty.hazelcast.session.SessionDataSerializer - sessions may not serialize correctly");
                     }
                     config.setInstanceName(hazelcastInstanceName);
+                    if (autoDetectionConfigDisabled) config.getNetworkConfig().getJoin().getAutoDetectionConfig().setEnabled(false);
```

Comment:
I don't really know what the ramification is of defaulting this to `disabled`? I assume we've historically had this `enabled`, and this is just a point release, so might be a surprise to users? Or is this a new parameter in version 5?